api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3311,4 +3314,10 @@ public void mergePatients_shouldFailIfNotPreferredPatientHasUnvoidedOrders() thr
 		Patient notPreferredPatient = patientService.getPatient(7);
 		patientService.mergePatients(preferredPatient, notPreferredPatient);
 	}
+
+	
+	@Test(expected = APIException.class)
+	public void processDeath_shouldThrowAPIExceptionIfPatientIsNull() throws Exception{
+		patientService.processDeath(null, new Date(), new Concept(), "unknown");
```

Comment:
Of recent i have seen a pattern of different people trying to increase test coverage by mostly testing for null arguments. Am now discouraging that by requiring to at least include some business logic testing, before i can merge.